vent/api/plugins.py

```diff
@@ -299,7 +366,7 @@ def _build_tools(self, status):
         # !! TODO implement features: wild, remove_old, disable_old, limit_groups
 
         # check result of clone, ensure successful or that it already exists
-        if status in [0, 128]:
+        if status:
```

Comment:
I don't believe so. I think I removed all the code that passes in statuses within that explicit range. I believe status are now only ever passed to _build_tools with values True/False. Did a quick grep on _build_tools to see all calls to it. Test case was the last one that passed in a numeric value.